test/tests/meshgenerators/lower_d_block_generator/tests

```diff
@@ -26,22 +26,13 @@
                   "existing sideset for"
     issues = '#13814'
     design = Mesh/index.md
-
-    [first_order]
-      type = Exodiff
-      exodiff = 'lower_d_out.e'
-      input = 'lower_d.i'
-      mesh_mode = REPLICATED
-
-      detail = "first-order and"
-    []
     [second_order]
       type = Exodiff
       exodiff = 'lower_d_second_order_out.e'
-      input = 'lower_d.i'
+      input = 'ids.i'
       mesh_mode = REPLICATED
       cli_args = 'Mesh/second_order=true Outputs/file_base=lower_d_second_order_out '
-                 'GlobalParams/order=SECOND Mesh/gen/nx=5 Mesh/gen/ny=5'
+                 'GlobalParams/order=SECOND Mesh/gmg/nx=5 Mesh/gmg/ny=5'
 
       detail = "second-order elements."
```

Comment:
With the removal of the first order test, we no longer need the nested testing block. You can combine into the `requirement` on line 25